src/styles/main.scss

```diff
@@ -1,3 +1,5 @@
-body {
-  margin: 0;
-}
+@use "utils/reset" as *;
```

Comment:
For the current version of the scss we are using, please use @import everywhere, example:
```
@use "utils/reset";
```